frontend/pages/events.js

```diff
@@ -90,13 +91,10 @@ const Events = () => {
     // show loading signs for past events
     setLoadingPast(true);
     const fetchPastEvents = async () => {
-      const res = await axios.get(
-        "https://wit-database.herokuapp.com/past-events"
+      const allEvents = allPastEvents.filter(
+        (event) => event.fields.year === Number(year)
       );
-      const allEvents = humps
-        .camelizeKeys(res.data)
-        .filter((event) => event.year === Number(year));
-      setTerms(allEvents.reverse());
+      setTerms(allEvents);
```

Comment:
Reimplement the loading element when user changes the timeline year

Reply:
added buffer of 500ms